astropy/visualization/wcsaxes/core.py

```diff
@@ -27,14 +26,7 @@
 VISUAL_PROPERTIES = ["facecolor", "edgecolor", "linewidth", "alpha", "linestyle"]
 
 if HAS_PIL:
-    from PIL.Image import Image
-
-    if minversion("PIL", "9.1"):
-        from PIL.Image import Transpose
-
-        FLIP_TOP_BOTTOM = Transpose.FLIP_TOP_BOTTOM
-    else:
-        from PIL.Image import FLIP_TOP_BOTTOM
+    from PIL.Image import Image, Transpose
```

Comment:
Hmm we don't actually list pillow anywhere, even in "all". Was this an oversight? 🤔

Reply:
PIL gets installed as a matplotlib dependency.